pennylane/ops/op_math/controlled_ops.py

```diff
@@ -339,6 +344,242 @@ def compute_matrix():  # pylint: disable=arguments-differ
     def _controlled(self, wire):
         return qml.CCZ(wires=wire + self.wires)
 
+    @staticmethod
+    def compute_decomposition(wires):  # pylint: disable=arguments-differ
+        return [qml.ControlledPhaseShift(np.pi, wires=wires)]
+
+    def decomposition(self):
+        return self.compute_decomposition(self.wires)
+
+
+def _check_and_convert_control_values(control_values, control_wires):
+    # TODO: maybe at some point actually deprecate string representations of control values?
```

Comment:
[scheduled](https://app.shortcut.com/xanaduai/story/49875/deprecate-passing-a-string-for-control-values-to-multicontrolledx) already for 0.36, so we can remove this TODO